src/components/Owner/CustomerEntry.jsx

```diff
@@ -31,11 +31,14 @@ const CustomerEntry = ({ customer }) => {
                             {`${customerData.first_name} ${customerData.last_name}`}
                         </Typography>
                     </Grid>
-                    <Grid item container direction="column" justify="center" alignItems="center">
+                    <Grid item container direction="column" justify="center" alignItems="center" xs={3}>
```

Comment:
all rendered jsx should be moved to separate component and instantiated here with props passed down